docs/apache-airflow/core-concepts/params.rst

```diff
@@ -220,7 +220,7 @@ The following features are supported in the Trigger UI Form:
     If no ``title`` is defined the parameter name/key is used instead.
   - The :class:`~airflow.models.param.Param` attribute ``description`` is rendered below an entry field as help text in gray color.
     If you want to provide special formatting or links you need to use the Param attribute
-    ``description_md``. See tutorial DAG ``example_params_ui_tutorial`` for an example.
+    ``description_md``. See tutorial DAG `example_params_ui_tutorial <https://github.com/apache/airflow/blob/main/airflow/example_dags/example_params_ui_tutorial.py>`_ for an example.
```

Comment:
I don't think that using direct links to the docs in the development branch is a good idea, as:
1. At some point these links might be relocated and become broken, without notice.
2. Instead of linking to another page, it would be easier if the relevant parts of the referred would in the same page as the docs.

Therefore, I'd suggest instead using the ``exampleinclude`` template - it exists in many `.rst` files, for example: [dags.rst](https://github.com/apache/airflow/blob/ae65820aec0e44fba1b0256821ab89bfd679bc58/docs/apache-airflow/core-concepts/dags.rst?plain=1#L290). If at some point we relocate these files, the CI will alert that it cannot find these files. Keep in mind that you might need to create sections in the example DAG file to use the `:start-after:` and `:end-before:` attributes for easier reading.

The same comment goes for the other links below.

Reply:
@shahar1 Thank you for your suggestion. Working on it!

Reply:
Well done, LGTM!